.github/workflows/ci-tk.yaml

```diff
@@ -59,9 +59,10 @@ jobs:
           pip install --no-cache-dir -r requirements-iree-pinned.txt --upgrade
           pip install -r requirements.txt -e .
 
-      - name: Install GPU pip deps
+      - name: Install GPU rocm and pip deps
         if: "(contains(toJSON(matrix.os), 'amdgpu') || contains(toJSON(matrix.os), 'mi300')) && !cancelled()"
         run: |
+          sudo apt install -y rocm
```

Comment:
Wouldn't the dockers already come with rocm? if not how are we running the e2e test rn?

Reply:
Yes good question. They should come with rocm but they dont. I think some parts of rocm are installed but not the entire suite of apps. Once we switch to docker images with all of rocm installed, we shouldn't need this ( I was surprised that we did ).

Reply:
For education purposes, do you know which things we are missing, maybe we can fix it at the root cause in the future

Reply:
hipsparse was the first one missing.